pkg/skaffold/deploy/kubectl/labels.go

```diff
@@ -40,12 +38,14 @@ func (l *ManifestList) SetLabels(labels map[string]string) (ManifestList, error)
 }
 
 type labelsSetter struct {
+	runID  string
 	labels map[string]string
 }
 
-func newLabelsSetter(labels map[string]string) *labelsSetter {
+func newLabelsSetter(runID string, labels map[string]string) *labelsSetter {
 	return &labelsSetter{
 		labels: labels,
+		runID:  runID,
```

Comment:
nit: keep runId and labels always in the same order (in struct, func args, struct creation)

Reply:
thanks, fixed but lmk if you see it done wrong anywhere else